apps/bankingAPI_web/lib/bankingAPI_web/controllers/user_controller.ex

```diff
@@ -0,0 +1,32 @@
+defmodule BankingAPIWeb.UserController do
+  @moduledoc """
+  Actions related to the user resource.
+  """
+  use BankingAPIWeb, :controller
+
+  alias BankingAPI.Users
+  alias BankingAPI.Users.Inputs
+
+  alias BankingAPIWeb.InputValidation
+
+  def create(conn, params) do
+    with {:ok, input} <- InputValidation.cast_and_apply(params, Inputs.Create),
+         {:ok, user} <- Users.create_new_user(input) do
+      send_json(conn, 200, user)
+    else
+      {:error, %Ecto.Changeset{}} ->
+        msg = %{type: "bad_input", description: "Invalid input"}
+        send_json(conn, 400, msg)
+
+      {:error, :email_conflict} ->
+        msg = %{type: "conflict", description: "Email already taken"}
+        send_json(conn, 412, msg)
+    end
+  end
+
+  defp send_json(conn, status, body) do
+    conn
+    |> put_resp_header("content-type", "application/json")
+    |> send_resp(status, Jason.encode!(body))
```

Comment:
No geral a gente usa views pra definir os json que vão ser retornados ao invés de serializar o schema retornado. Isso porque geralmente a relação não é 1 pra 1. Mas pro teu caso aqui faz sentido ser só uma serialização do schema.

Reply:
Mesmo nesses casos (1 para 1), eu tenho que criar uma VIEW ou posso deixar dessa forma mesmo?

Reply:
se for 1 pra 1 pode deixar assim